lib/combinat.gi

```diff
@@ -2446,6 +2446,72 @@ InstallGlobalFunction( "IteratorOfPartitions", function( n )
     end );
 
 
+#############################################################################
+##
+#F  IteratorOfPartitionsSet( <set> )
+##
+##  all unordered partitions of the set <A>set</A> into pairwise disjoint
+##  nonempty sets.
+##
+InstallGlobalFunction( IteratorOfPartitionsSet, function( s )
+    local nextIterator, isDone, m, out, i, shallowCopy;
+
+      if not IsSet(s) then
+          Error( "<s> must be a positive integer" );
```

Comment:
This seems to be not the appropriate error message.

Reply:
I have corrected the error message. It should say "set" instead of "positive integer".